packages/shared/src/error.ts

```diff
@@ -6,6 +6,10 @@ export function isUnauthorizedError(e: any): boolean {
   return code === 'authentication_invalid' && status === 401;
 }
 
+export function isCaptchaError(e: ClerkAPIResponseError): boolean {
+  return ['captcha_invalid', 'captcha_not_enabled', 'captcha_missing_token'].includes(e.errors[0].code);
```

Comment:
```suggestion
  return ['captcha_invalid', 'captcha_not_enabled', 'captcha_missing_token'].includes(e?.errors?.[0]?.code);
```

Reply:
Not needed as `e`is `ClerkAPIResponseError` so these are already defined :)